dev/run-tests.py

```diff
@@ -583,6 +589,7 @@ def main():
     modules_with_python_tests = [m for m in test_modules if m.python_test_goals]
     if modules_with_python_tests:
         run_python_tests(modules_with_python_tests, opts.parallelism)
+        run_python_packaging_tests()
```

Comment:
+1 as well; this seems cheap to run and it's better to err on the side of running things more often.